From 1cc008fc8152daffaf1b23afc5f5713f5b457978 Mon Sep 17 00:00:00 2001
From: Manuel Oetiker <manuel@oetiker.ch>
Date: Sat, 28 Jul 2018 18:13:43 +0200
Subject: [PATCH] fix for openssl 1.1.N (#22)

* Update README.md

add make

* it should work with openssl 1.1.N to
---
 bin/acmefetch | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/acmefetch b/bin/acmefetch
index 5309ede..ee2553e 100755
--- a/bin/acmefetch
+++ b/bin/acmefetch
@@ -70,7 +70,7 @@ sub getDataProcessor {
                     default => 'openssl',
                     validator => sub {
                         my $bin = shift;
-                        `$bin version 2>&1` =~ /^OpenSSL 1.0/ or return "Openssl $bin not found";
+                        `$bin version 2>&1` =~ /^OpenSSL 1/ or return "Openssl $bin not found";
                         return "";
                     }
                 },
